source/app/components/ExportPreview/ExportPreview.js

```diff
@@ -0,0 +1,165 @@
+import {
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalBody,
+  Box,
+  Flex,
+  Button,
+} from '@chakra-ui/react';
+import { connect } from 'react-redux';
+import queryString from 'query-string';
+import { ExportTypes, downloadRedactedDocument } from '../../utils/downloadRedactedDocument';
+import { getDocumentById } from '../../store/entities/documents/selectors';
+import { getDocumentPageCount } from '../../utils/document';
+import { Document, Page } from 'react-pdf';
+import Loading from '../Loading/Loading';
+import { DocumentMarks, useDocumentResizer } from '../DocumentViewer/DocumentViewer';
+import DownloadIcon from './DownloadIcon';
+
+const ExportPreview = ({ document }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { containerRef, documentWidth, handleResize } = useDocumentResizer(true, []);
+  const [isExporting, setIsExporting] = React.useState(false);
+
+  const readableDocumentName = document.documentName.replace(/\.(pdf|PDF|png|PNG|jpg|JPG|jpeg|JPEG)/, '');
+  const pageCount = getDocumentPageCount(document);
+
+  const downloadRedactedDocumentWithDownloadingState = async (exportType) => {
+    try {
+      setIsExporting(true);
+      await downloadRedactedDocument(document, exportType);
+    } finally {
+      setIsExporting(false);
+    }
+  };
+
+  return (
+    <>
+      <Button
+        onClick={onOpen}
+        size='sm'
+        borderRadius='2px'
+        fontSize='1rem'
+        bg='orange'
+        _hover={{ background: '#f6761d' }}
+        _active={{ background: '#f6761d' }}
+      >
+        Export Redacted Doc
```

Comment:
Do we want to provide an option to "Export Redacted Doc" when there are no active Redactions?
Could we have it on a conditional similar to how we enable Clear Redactions?

Reply:
We could but I didn't feel that it was necessary to make it conditional. Making the clear button conditional adds clarity to the user that clicking the button would not have done anything anyway. I don't see a similar reason for why the user should be disallowed from exporting a document without any redactions.

Reply:
The reason I bring it up is because its more of an "Export Doc" rather than "Export Redacted Doc". 
The preview page is also called "Preview Redacted Doc" and has the option to "Download Redacted Doc"
I am wondering if there is actually a use case for the user to "just convert" the doc in DUS and that too in Compliance track or it would just add confusion

Reply:
Discussed offline. The redaction will continue to stay based on the condition